빌드 시 Spotbugs `SS_SHOULD_BE_STATIC`

```sh
[ERROR] Medium: Unread field: io.jenkins.plugins.naverworks.bot.message.Action.type; should this field be static? [io.jenkins.plugins.naverworks.bot.message.Action] At Action.java:[line 13] SS_SHOULD_BE_STATIC
[INFO]


To see bug detail using the Spotbugs GUI, use the following command "mvn spotbugs:gui"
```

--- a/src/main/java/io/jenkins/plugins/naverworks/bot/message/Action.java
+++ b/src/main/java/io/jenkins/plugins/naverworks/bot/message/Action.java
@@ -10,7 +10,7 @@ public class Action {
     /**
      * uri로 고정
      */
-    private final String type = "uri";
+    private static final String TYPE = "uri";
 
     /**
      * 항목에 표시되는 레이블
@@ -34,7 +34,7 @@ public class Action {
     }
 
     public String getType() {
-        return type;
+        return TYPE;
     }
 
     public String getLabel() {

--- a/src/test/java/io/jenkins/plugins/naverworks/bot/message/carousel/CarouselContentTest.java
+++ b/src/test/java/io/jenkins/plugins/naverworks/bot/message/carousel/CarouselContentTest.java
@@ -1,9 +1,8 @@
-package io.jenkins.plugins.naverworks.bot.message;
+package io.jenkins.plugins.naverworks.bot.message.carousel;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import io.jenkins.plugins.naverworks.bot.message.carousel.CarouselContent;
-import io.jenkins.plugins.naverworks.bot.message.carousel.Column;
+import io.jenkins.plugins.naverworks.bot.message.MessageFixture;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
@@ -34,7 +33,7 @@ class CarouselContentTest {
 
         // then
         String actual = objectMapper.writeValueAsString(columns);
-        String expected = "[{\"originalContentUrl\":null,\"field\":null,\"title\":\"MARK-1\",\"text\":\"Jira Issue 1\",\"defaultAction\":{\"type\":\"uri\",\"label\":null,\"uri\":null},\"actions\":[{\"type\":\"uri\",\"label\":\"more\",\"uri\":\"https://markruler.github.io/\"}]}]";
+        String expected = "[{\"originalContentUrl\":null,\"field\":null,\"title\":\"MARK-1\",\"text\":\"Jira Issue 1\",\"defaultAction\":{\"label\":null,\"uri\":null,\"type\":\"uri\"},\"actions\":[{\"label\":\"more\",\"uri\":\"https://markruler.github.io/\",\"type\":\"uri\"}]}]";
         assertThat(actual).isEqualTo(expected);
     }
 

--- a/src/test/java/io/jenkins/plugins/naverworks/bot/message/list/ListTemplateContentTest.java
+++ b/src/test/java/io/jenkins/plugins/naverworks/bot/message/list/ListTemplateContentTest.java
@@ -1,9 +1,8 @@
-package io.jenkins.plugins.naverworks.bot.message;
+package io.jenkins.plugins.naverworks.bot.message.list;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import io.jenkins.plugins.naverworks.bot.message.list.Element;
-import io.jenkins.plugins.naverworks.bot.message.list.ListTemplateContent;
+import io.jenkins.plugins.naverworks.bot.message.MessageFixture;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
@@ -34,7 +33,7 @@ class ListTemplateContentTest {
 
         // then
         String actual = objectMapper.writeValueAsString(elements);
-        String expected = "[{\"title\":\"MARK-1\",\"subtitle\":\"Jira Issue 1\",\"action\":{\"type\":\"uri\",\"label\":\"more\",\"uri\":\"https://markruler.github.io/\"}}]";
+        String expected = "[{\"title\":\"MARK-1\",\"subtitle\":\"Jira Issue 1\",\"action\":{\"label\":\"more\",\"uri\":\"https://markruler.github.io/\",\"type\":\"uri\"}}]";
         assertThat(actual).isEqualTo(expected);
     }
 }

